### DISCUSSION TITLE
Project will be moving and split int subprojects

### DISCUSSION THREAD

**Discussion by Gcenx** [The-Wineskin-Project/WineskinServer#291] [Announcements]
The project will be moved under https://github.com/The-Wineskin-Project

WineskinServer will be split into `Wrapper` `Winery`, `Engines` And project renamed to `Wineskin`

The wiki will also be split off into a separate repository to make it easier for others to contribute.

**Comment by Rastafabisch** [comment]
Just a thought but as the project is outgrowing its small skin it could be renamed into wine cask or distillery or something alike. I don't think this will make a huge impact on discoverability, as the go to git will remain your gcenx top domain. 
That said, any further progress to report, as since the announcement there weren't further news.

**Reply by Gcenx** [reply]
The repo name will slightly change but the actual project lightly won’t be renamed or there may be confusion.

Honestly IRL has and still is rather chaotic slowing progress to a crawl, but the intention is stop to move everything over.

<br>

As the majority of the project is done solely by me it takes time to do things as correctly as possible.

I’d like to get this done around or before CX23 lands as I do plan on dropping even more legacy Engines along with the required dylibs.
